packages/ui/src/components/va-input/components/VaMessageList/VaMessageList.vue

```diff
@@ -9,25 +9,36 @@
       :key="index"
       class="va-message-list__message"
     >
-      {{ message }}
+      <va-icon
+        v-if="isError"
+        class="va-message-list__icon"
+        name="warning"
+        :size="16"
+      />{{ message }}
     </div>
   </div>
 </template>
 
 <script lang="ts">
 import { computed, defineComponent, PropType } from 'vue'
+
 import { useColors } from '../../../../composables'
 
+import { VaIcon } from '../../../va-icon'
+
 export default defineComponent({
   name: 'VaMessageList',
 
+  components: { VaIcon },
+
   props: {
     modelValue: {
       type: [String, Array] as PropType<string | string[]>,
       default: '',
     },
     limit: { type: Number, default: 1 },
     color: { type: String },
+    isError: { type: Boolean, default: false },
```

Comment:
Why it's named as `isError` instead of `hasError` or something similar?

Reply:
It's state naming and seems naturally to me. But `hasError` is also ok. Will rename.